src/net.h

```diff
@@ -673,15 +674,15 @@ class CNode
     const bool fInbound;
     std::atomic_bool fSuccessfullyConnected;
     std::atomic_bool fDisconnect;
+    mutable CCriticalSection cs_filter;
```

Comment:
Is moving those lines required? It makes automated review harder, so best to keep them where they were?

Reply:
Good point! Fixed. Please re-review :-)